test/javascripts/acceptance/codebytes-test.js

```diff
@@ -44,9 +46,8 @@ acceptance("CodeBytes", function (needs) {
       '[codebyte]\n\n[/codebyte]'
     );
 
-    assert.equal(
-      queryAll(".d-editor-preview .d-codebyte iframe").attr('src'),
-      "https://www.codecademy.com/codebyte-editor?lang=&text=",
+    assert.ok(
+      queryAll(".d-editor-preview .d-codebyte iframe").attr('src').startsWith("https://www.codecademy.com/codebyte-editor"),
```

Comment:
might be a bit more semantic to do an `assert.match` with a regex here, assuming that works the same

Reply:
I looked into your suggestion but unfortunately this is `QUnit.assert` which does not have a `match` function :(

Reply:
woops totally forgot it was qunit not jest, sorry bout that!